CODEOWNERS

```diff
@@ -153,6 +153,8 @@
 /drivers/serial/uart_liteuart.c           @mateusz-holenko @kgugala @pgielda
 /drivers/serial/Kconfig.rtt               @carlescufi @pkral78
 /drivers/serial/uart_rtt.c                @carlescufi @pkral78
+/drivers/serial/Kconfig.xlnx              @wjliang
```

Comment:
Btw, @bbolen would you like to add someone in the list of maintainers for the arch/arm/core/cortex-r ?

Reply:
Added myself.